tests/integration/delete_user.t

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/perl -w
+
+use ProductOpener::PerlStandards;
+
+use Test::More;
+use ProductOpener::APITest qw/:all/;
+use ProductOpener::Test qw/:all/;
+use ProductOpener::TestDefaults qw/:all/;
+use ProductOpener::Users qw/:all/;
+use ProductOpener::Producers qw/:all/;
+
+remove_all_users();
+wait_application_ready();
+
+#new common user agent
+my $ua = new_client();
+my %create_client_args = (%default_user_form, (email => 'bob@test.com'));
+create_user($ua, \%create_client_args);
+
+#new admin user agent
+my $admin = new_client();
+create_user($admin, \%admin_user_form);
+
+#common ua add a new product then delete the account while being still logged in
+edit_product($ua, \%default_product);
+
+my @words = (
```

Comment:
It's interesting pattern (yeah I can see how it was interesting to copy paste the same line all over) but it hinders readability of your test, and it should be favored.

So I would replace @words[1] etc... by the string instead.

Reply:
Okay, yes it's more readable, I'll change this